### DISCUSSION TITLE
sample_count and debounce_delay_ms in Drum::Config

### DISCUSSION THREAD

**Discussion by HellbroWDR** [ravinrabbid/DonCon2040#2] [Q&A] [answered]
In your code in Drum.cpp, you sample input sample_count times and use debounce_delay_ms to maintain time lag of triggering hits, but I can’t find them in drum_config in include/GlobalConfiguration.h. Could you tell me how you set them, please?

**Comment by ravinrabbid** [comment, marked as answer]
Sorry, I forgot to push them. Should be there now.
